Dynatrace/dynatrace-operator
# Allow duplicate oneagent arguments

## Description

With a previous overhaul of how OneAgent arguments are handled it was made impossible to use an argument multiple times. For some arguments that's reasonable, for others not so much.

This PR enables using the same argument multiple times for the OneAgent, while keeping some more sensible arguments to being used only once. Only those where the operator potentially uses default values are considered. Namely those which can't be used multiple times are:
* `--set-server`
* `--set-tenant`
* `--set-host-id-source`
* `--set-host-group`
* `--set-network-zone`
* `--set-proxy`

## How can this be tested?

Deploy the operator.
Deploy a DynaKube where you set custom one agent arguments. Use especially multiple `--set-host-property` and `--set-host-tag` entries. They should be usable multiple times.

## Checklist
